customs.js

```diff
@@ -55,6 +57,13 @@
       }
     )
     .then(
+      (function (result) {
+        if (result.lockout) {
+          return this.db.lockAccount(account)
+        }
+      }).bind(this)
+    )
+    .then(
       function () {},
       function (err) {
         log.error({ op: 'customs.flag.1', email: email, err: err })
```

Comment:
(Looks like GitHub won't let me comment on a line that is below, this diff, so this comment refers to the (1) and (2) comments below here.)

And now (3) if the `this.db.lockAccount(account)` fails too.

As above, I'd almost prefer if this returned the `result` to the calling function, which then checks `db.lockAccount()`. I guess this all works, but am just thinking about who should be responsible calling this and I think the higher level `routes/account.js` should make that decision. I'm not 100% flat out "it has to be" on this but I think it'd make for a nicer separation.